idseq_pipeline/commands/common.py

```diff
@@ -662,10 +664,11 @@ def upload_commit_sha(version, s3_destination=None):
 
     # also initialize the main version output file with the commit sha and the job_id
     global OUTPUT_VERSIONS
-    aws_batch_job_id = os.environ.get('AWS_BATCH_JOB_ID', 'local')
```

Comment:
Did you remove this line by accident?

Reply:
@cdebourcy no it's the same as line 657 || 659